Bottom: f72e404022cc3892be6a2722d337af2cab14e9fd
Top:    4b810000b83b48e6ea63cba940ba9490dfaba4e9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 23:40:01 +0100

hid/lesstif: Remove unused variable in use_mask routine


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 776a345..cdaa7b0 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -3039,8 +3039,6 @@ lesstif_destroy_gc (hidGC gc)
 static void
 lesstif_use_mask (int use_it)
 {
-  static Window old;
-
   if ((TEST_FLAG (THINDRAWFLAG, PCB) || TEST_FLAG(THINDRAWPOLYFLAG, PCB)) &&
       !use_xrender)
     use_it = 0;
